datahub/company/test/test_tasks.py

```diff
@@ -88,15 +88,19 @@ def test_lock(
     @freeze_time('2020-01-01-12:00:00')
     def test_no_interactions(
         self,
-        caplog,
+        monkeypatch,
         automatic_company_archive_feature_flag,
         simulate,
     ):
         """
         Test that a company without interaction that fits
         all the other criteria is archived.
         """
-        caplog.set_level(logging.INFO, logger='datahub.company.tasks')
+        mock_log_to_sentry = mock.MagicMock()
```

Comment:
Is MagicMock needed here?

```suggestion
        mock_log_to_sentry = mock.Mock()
```